src/geofabrics/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 @author: pearsonra
 """
-__version__ = "0.3.0"
\ No newline at end of file
+__version__ = "0.4.0"
```

Comment:
Is there a check you can add to the GitHub Actions to bump versions or perform the check. I would never remember to change this

Reply:
I think I will leave it for now - but your suggestion of using GH actions is great. We could tie it to the associated PR/branch milestone. I'll look into this before we go to release